File: BtgChallenge/BtgChallenge/Networking/Response/LiveResponse.swift
//
//  LiveResponse.swift
//  BtgChallenge
//
//  Created by Felipe Alexander Silva Melo on 13/05/20.
//  Copyright © 2020 Felipe Alexander Silva Melo. All rights reserved.
//

import Foundation

struct LiveResponse: Codable {
    let success: Bool
    let terms, privacy: String
    let timestamp: Int
    let source: String
    let quotes: [String: Double]
}

File: app/src/main/java/com/todeschini/currencyconverter/model/LiveResponse.kt
package com.todeschini.currencyconverter.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class LiveResponse (
    @SerializedName("success")
    val success: Boolean,
    @SerializedName("terms")
    val terms: String,
    @SerializedName("privacy")
    val privacy: String,
    @SerializedName("timestamp")
    val timestamp: Long,
    @SerializedName("source")
    val source: String,
    @SerializedName("quotes")
    val quotes: Map<String, Double>
): Serializable
